docs: add links to our migration skills (#281)
NomicFoundation/hardhat-website: src/content/docs/docs/migrate-from-hardhat2/index.mdx

import Install from "@hh/Install.astro";
import Run from "@hh/Run.astro";

:::tip

> If you use an AI coding agent, you can install our Hardhat 2 to Hardhat 3 migration skill to have it perform the migration for you:

Reviewer: Same as the above comment https://github.com/NomicFoundation/hardhat-website/pull/281/changes#r3474147042. Here I' d stress it even more, since a full project migration is more complex then a "simple" foundry to hh migration

Author: ```suggestion
If you use an AI coding agent, you can install our Hardhat 2 to Hardhat 3 migration skill to have it aid your migration:
```
What do you think?

Author: Applied here: https://github.com/NomicFoundation/hardhat-website/pull/281/changes/23c678a6de75a4951f6f19e1c5a7d98c38b237a9
